src/app.py

```diff
@@ -3,6 +3,7 @@
 from flask import Flask, jsonify
 from flask_cors import CORS
 from flask_restful import Api
+from apscheduler.schedulers.background import BackgroundScheduler
```

Comment:
Thank you for bringing that up. I didn't know celery has support for scheduled tasks until I peeked at theirs [docs](http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html) moments. I'll begin work on refactoring this feature to use celery immediately. Meanwhile, this can be merged if I still haven't delivered something before the MVP.